README.md

```diff
@@ -29,6 +25,104 @@ and model-benchmark to measure domain-aware end-to-end deep learning workloads.
 SuperBench is in the early pre-alpha stage for open source, and not ready for general public yet.
 If you want to jump in early, you can try building latest code yourself.
 
+## SuperBench capabilities, workflow and benchmarking metrics
+
+The following graphic shows the capabilities provide by SuperBench core framework and its extension.
+
+<img src="imgs/superbench_structure.png">
+
+Benchmarking metrics provided by SuperBench are listed as below.
+
+<table>
+  <tbody>
+    <tr align="center" valign="bottom">
+      <td>
+      </td>
+      <td>
+        <b>Micro Benchmark</b>
+      </td>
+      <td>
+        <b>Model Benchmark</b>
+      </td>
+    </tr>
+    <tr valign="top">
+      <td align="center" valign="middle">
+        <b>Metrics</b>
+      </td>
+      <td>
+        <ul><li><b>Computation Benchmark</b></li>
+          <ul><li><b>Kernel Performance</b></li>
+            <ul>
+              <li>GFLOPS</li>
+              <li>TensorCore</li>
+              <li>cuBLAS</li>
+              <li>cuDNN</li>
+            </ul>
+          </ul>
+          <ul><li><b>Kernel Launch Time</b></li>
+            <ul>
+              <li>Kernel_Launch_Event_Time</li>
+              <li>Kernel_Launch_Wall_Time</li>
+            </ul>
+          </ul>
+          <ul><li><b>Operator Performance</b></li>
+            <ul><li>MatMul</li><li>Sharding_MatMul</li></ul>
+          </ul>
+          <ul><li><b>Memory</b></li>
+            <ul><li>H2D_Mem_BW_&lt;GPU ID&gt;</li>
+              <li>H2D_Mem_BW_&lt;GPU ID&gt;</li></ul>
```

Comment:
```suggestion
            <ul><li>H2D_Mem_BW_<GPU ID></li>
              <li>H2D_Mem_BW_<GPU ID></li></ul>
```

Reply:
It is he way to show the "<" and ">" in html, or it will be hidden.